setup.py

```diff
@@ -1,6 +1,7 @@
 from setuptools import setup, find_packages
 
-version = "2.10.3"
+with open("metaflow/version.py", mode="r") as f:
```

Comment:
was there some reason why `version.py` couldn't be imported here instead of resorting to string parsing? This seems to work for me at least, does it have some unwanted side-effects?

```python
from metaflow.version import metaflow_version

version = metaflow_version
```

Reply:
Am fine with that. I just didn't want to "import" a package being built. I wasn't sure if there would be issues with it but if that works well, we can do that too.